eventlogger/index.js

```diff
@@ -17,7 +18,13 @@ EventLogger.prototype.watch = function (obj, options) {
 
   //stupid way of identify a hapi server
   if (obj.app && obj.info && obj.info.id && obj.info.created) {
-    return hapi_server.watch(this.logger, obj, options);
+    const hapiVersion = obj.version || ""; // '17.0.0'
```

Comment:
I'm still ambivalent about this multiple version support, but seems to be the way the lib is doing it for other tools, so ok.

Reply:
Will be better to have one _hapi_ plugin that's uses `auth0-common-logging` that gets updated. But i think that a factory method is a reasonable way to solve it.